### PR TITLE
[bitnami/redis] Fix clustering when service port != container port

### PR DESCRIPTION
**Pull request by javsalgar**
**Description of the change**

The clustering was configured using the headless service (that will return the pod IPs) with the service IPs (which is not present in the containers). This was causing conflicts when changing the service port. This PR fixes this. 

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Review by marcosbc — APPROVED** [review]
LGTM!

File: bitnami/redis/templates/scripts-configmap.yaml [hunk 1]
```diff
@@ -241,20 +241,19 @@ data:
...
         sleep {{ .Values.sentinel.cleanDelaySeconds }}
     done
     info "Sentinels clean up done"
 
-
     sentinel_conf_set "sentinel monitor" "{{ .Values.sentinel.masterSet }} "$REDIS_MASTER_HOST" "$REDIS_MASTER_PORT_NUMBER" {{ .Values.sentinel.quorum }}"
 
     add_replica() {
         if [[ "$1" != "$REDIS_MASTER_HOST" ]]; then
-            sentinel_conf_add "sentinel known-replica {{ .Values.sentinel.masterSet }} $1 {{ .Values.sentinel.service.port }}"
```
Comment (dannylee-) [inline bitnami/redis/templates/scripts-configmap.yaml:257]: Is this wrong?  Shouldn't this be .Values.sentinel.service.port?

Comment (dannylee-) [reply bitnami/redis/templates/scripts-configmap.yaml:257]: Or maybe it should .Values.replica.containerPort

Comment (marcosbc) [reply bitnami/redis/templates/scripts-configmap.yaml:257]: Previously, `sentinelPort` was set to `26379` by default, using the same value as `sentinel.containerPort`. In that sense, I don't think this has changed value. Are you finding any particular issues?

Comment (dannylee-) [reply bitnami/redis/templates/scripts-configmap.yaml:257]: The port that is set in the "sentinel known-replica" used to be 6379.  After I got this change, the port changed to 26379, which caused a bunch of weird issues (i.e. somehow in a 3 pod setup, the sentinel container in one of the redis pods thought there were 4 sentinels somehow).  Once I changed it back to 6379, everything worked as expected.

**Pull request merged**